nextgen/project-expander/src/tasks_expander.rs

```diff
@@ -159,11 +164,14 @@ impl<'graph, 'query> TasksExpander<'graph, 'query> {
                     if dep_target.task_id == task.id {
                         // Avoid circular references
                     } else {
-                        check_and_push_dep(project, dep, false)?;
+                        check_and_push_dep(project, dep, dep.skip_if_missing.unwrap_or(false))?;
                     }
                 }
                 // id:task
                 TargetScope::Project(project_locator) => {
+                    if dep.skip_if_missing.is_some() {
+                        // log a message to the user to let them know that this is effectless
```

Comment:
I think on line 185 you'll need to handle the error case, because it won't exist sometimes.

Reply:
Not sure, I just wanted to log a message (not changing this case) 🤔  (line 175 and 189 are left unchanged)
Except if we want to allow skipping project dependencies 🤔